### PR TITLE
Cleanup of the diagnostics section.

### PR DESCRIPTION
**Pull request by cbrueffer**
### Description of the proposed additions or changes

Various cleanups:
- spelling/grammar
- whitespace
- acronyms

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -2,81 +2,84 @@
...
-##### qRT-tPCR
+##### qRT-PCR
 
-Around the same time, researchers in China developed a Quantitative Real Time Reverse Transcription PCR (qRT-tPCR) test to identify two region of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029]. 
-Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses. 
-The assay uses the N gene to screen patients, while the orf1b is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
-In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2 is one. 
-Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2. 
-However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade. 
+Around the same time, researchers in China developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two region of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
```
Comment (agitter) [inline content/10.diagnostics.md:25]: https://www.ncbi.nlm.nih.gov/probe/docs/techqpcr/ lists "real-time quantitative reverse transcription".  I'm not sure what the usual convention is.

Comment (cbrueffer) [reply content/10.diagnostics.md:25]: Indeed, both descriptions seem to be in use. The same goes for the acronym, qRT-PCR and RT-qPCR. Not sure what the preferred version is.

Comment (agitter) [reply content/10.diagnostics.md:25]: Because I'm not an authority, let's go with the version you used.

**Comment by rando2** [comment]
Thanks for cleaning this up @cbrueffer!

**Pull request merged**